### ISSUE TITLE
Bug appears when adding experimen to a new project

### ISSUE THREAD

**Issue by drmackay** [ParaToolsInc/taucmdr#29]
Created a new project as follows --
tau project create 4mpi
 tau select 4mpi
tau project edit 4mpi --add-measurements=profile-sample
tau project edit 4mpi --add-application=samplematrixcode
 tau project edit 4mpi --add-target=delphi
tau select

(typed tau select to get advice on adding experiment to project as it still had no experiment - that's when bug exposed)
[debug_log2.txt](https://github.com/ParaToolsInc/taucmdr/files/138728/debug_log2.txt)
[dash.txt](https://github.com/ParaToolsInc/taucmdr/files/138729/dash.txt)

**Comment by jlinford** [comment]
Reproduced:
```
tau init
tau proj cre 4mpi
tau proj ed 4mpi --add-measurements profile sample
tau proj ed 4mpi --add-application mm
 tau proj ed 4mpi --add-target east03
tau project select 4mpi
tau dash  # Shows correct info, but experiments looks strange
```

This should create a new experiment:
```
tau sel sample
[TAU] CRITICAL
[TAU] 
[TAU] There is no experiment named 'east03-mm-sample' in project '4mpi'.
[TAU] 
[TAU] Hint: Try `tau --help`
[TAU] 
[TAU] TAU cannot proceed with the given inputs.
[TAU] Please check the selected configuration for errors or contact <support@paratools.com> for assistance.
```

This should create a new experiment:
```
tau exper cre east03 mm sample


[TAU] CRITICAL
[TAU] 
[TAU] An unexpected UniqueAttributeError exception was raised:
[TAU] 
[TAU] Experiment: A record with one of 'name=east03-mm-sample' already exists
[TAU] 
[TAU] Traceback (most recent call last):
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/tau", line 48, in <module>
[TAU]     sys.exit(cli_main_cmd.main(sys.argv[1:]))
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/../packages/taucmdr/cli/commands/__main__.py", line 133, in main
[TAU]     return cli.execute_command([cmd], cmd_args)
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/../packages/taucmdr/cli/__init__.py", line 296, in execute_command
[TAU]     return main(cmd_args or [])
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/../packages/taucmdr/cli/cli_view.py", line 87, in main
[TAU]     return cli.execute_command([args.subcommand], args.options, self.module_name)
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/../packages/taucmdr/cli/__init__.py", line 296, in execute_command
[TAU]     return main(cmd_args or [])
[TAU]   File "/home/jlinford/workspace/taucmdr/bin/../packages/taucmdr/cli/commands/experiment/create.py", line 142, in main
[TAU]     raise UniqueAttributeError(Experiment, "name="+name)
[TAU] UniqueAttributeError
[TAU] 
[TAU] This is a bug in TAU Commander.
[TAU] Please use --log to enable debug logging and send '/home/jlinford/.tau/debug_log' to <support@paratools.com> for assistance.
```

**Comment by khsa1** [comment]
Issue resolved as of 12eb3b5.
